Sources/Classes/core/Protocols/MiniAppUserInfoDelegate.swift

```diff
@@ -72,13 +72,13 @@ public extension MiniAppUserInfoDelegate {
 }
 
 /// Contact Object of a User
-public class MAContact: Codable, Equatable, Hashable {
+public class MAContact: Codable, Equatable, Hashable, Identifiable {
     /// Contact ID
-    public let id: String
+    public var id: String
```

Comment:
Any reason why we need this as `var`

Reply:
to change the user properties with bindings in swiftui (eg. `MiniAppSettingsContactsView`) imho the `MAUser` was public before so we can change the properties anyways just in a much more complicated way. We could also create a new `MAContact` and convert it back and forward just for the sample app but any reasons it should not be easily modifable?